cli/src/commands/create/dataset.rs

```diff
@@ -63,7 +63,7 @@ pub fn create(client: &Client, args: &CreateDatasetArgs) -> Result<()> {
                 title: title.as_ref().map(|title| title.as_str()),
                 description: description.as_ref().map(|description| description.as_str()),
                 has_sentiment: *has_sentiment,
-                entity_kinds: &entity_kinds,
+                entity_defs: entity_defs,
```

Comment:
Clippy failing here with lint to use shorthand:

```suggestion
                entity_defs,
```